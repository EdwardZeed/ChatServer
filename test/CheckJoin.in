REGISTER test4 ABC
LOGIN test4 ABC
CREATE channel3
JOIN channel3
q