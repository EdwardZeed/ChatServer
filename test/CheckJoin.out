RESULT REGISTER 1

RESULT LOGIN 1

RESULT CREATE channel3 1

RESULT JOIN channel3 1

